File: Modul 6/Soal 3/PRAK603-2410817210006-Andre_Cristian_Nathanael.c
#include <stdio.h>

int main() {
    int a, b, angka, i;
    scanf("%d %d", &a, &b);
    if (a != b) {
        printf("Jumlah tidak sama");
    } else {
        int barisa[a];
        int barisb[b];
        for (i = 0; i < a; i++) {
            scanf("%d", &barisa[i]);
        }
        for (i = 0; i < b; i++) {
            scanf("%d", &barisb[i]);
        }
        for (i = 0; i < a; i++) {
            angka = barisa[i] * barisb[i];
            printf("%d ", angka);
        }
    }
}

File: Modul 6/Soal 3/PRAK603-2410817210006-Andre_Cristian_Nathanael.py
a, b = map(int, input().split())
if (a != b):
    print('Jumlah tidak sama')
else:
    baris_a = list(map(int, input().split()))
    baris_b = list(map(int, input().split()))
    for i in range (a):
        print(baris_a[i] * baris_b[i], end=' ')